frontend/readme.md

```diff
@@ -1,16 +1,38 @@
-#Social Record FrontEnd  
-The frontend for Social Record
+## Social Record front-end  
+The Social Record front-end displays the analyzed data from instagram of the visitor for the exhibition. Using the front-end the visitors can experience what public data social media providers expose about them.
```

Comment:
`of the visitor for the exhibition` -> `of the visitor of the exhibition`

Reply:
changed it.